src/main/controllers/cmc-service.js

```diff
@@ -10,32 +10,10 @@ const EventEmitter = require('events');
 
 module.exports = function (app) {
     const eventEmitter = new EventEmitter();
-  
-    const sortCMCData = (data) => {
-        return data.sort((a, b) => {
-            let symbolA = a.symbol.toLowerCase();
-            let symbolB = b.symbol.toLowerCase();
-            if (symbolA == 'eth') {
-                return -1;
-            }
 
-            if (symbolB == 'eth') {
-                return 1;
-            }
 
-            if (symbolA == 'key') {
-                return -1;
-            }
-
-            if (symbolB == 'key') {
-                return 1;
-            }
-
-            return 0; //the others is not important
-        });
-    };
     const loadCmcData = () => {
-        request.get(config.cmcUrl, (error, httpResponse, result) => {
+        return request.get(config.cmcUrl, (error, httpResponse, result) => {
```

Comment:
just out of curiosity and benefit of learning, whats the thought process / benefit behind adding return to everything?  it makes things more explicit or does it actually change behaviour of the code? thanks!

Reply:
@altninja It stopped the warnings on the console. cheers

Reply:
lol nice ok thanks!